package.sh

```diff
@@ -16,6 +16,7 @@ Params:
     -c  Commit id of the git repository HEAD
     -s  Subdirectory to be packaging
     -g  Git version string
+    -k  Kernel configuration to be built
```

Comment:
Could this be just general "configuration" parameter, so that it could be used for other modules also if needed?

Reply:
Yes, of course. :) I will do the needed changes and remove the the word kernel from variables.